spec/features/visitors/i18n_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+feature 'Internationalization' do
+  context 'visit homepage' do
+    before do
+      page.driver.header 'Accept-Language', locale
+      I18n.locale = locale
```

Comment:
I think we want to drop this `I18n.locale = ` line because it would mask the work that the `Accept-Language` header should be doing